examples/arrays/arrays.metatags

```diff
@@ -1,2 +1,6 @@
-description: This BBE demonstrates arrays in Ballerina.
-keywords: ballerina, ballerina by example, bbe, arrays, mutable list
+description: Create a collection in Ballerina, Create an array, Create fixed length collection,
```

Comment:
```suggestion
description: Create a collection in Ballerina, Create an array, Create fixed length collection,
```
The description is too lengthy and needs to be summarized into one or two sentences.

Reply:
Discussed with @hasithaa @gimantha, and decided to keep this as this is to find a better way later.